py/fiberassign/assign.py

```diff
@@ -336,30 +336,33 @@ def write_assignment_fits_tile(asgn, fulltarget, overwrite, params):
                     # position in the hardware model.  Used this fixed fiber location.
```

Comment:
This is where we're writing out the results, and we're computing the positions we're going to "request" for a stuck or broken positioner.  This is irrelevant for actually placing fibers, of course, since they can't move anyway, but it does affect the output tables (and plots/QA).